lib/flat_search/filters/filter.ex

```diff
@@ -19,7 +19,6 @@ defmodule FlatSearch.Filters.Filter do
   def changeset(filter, attrs) do
     filter
     |> cast(attrs, @default_params)
-    |> validate_required(:max_price)
     |> validate_inclusion(:max_price, 1..100_000)
```

Comment:
btw. 
`validate_inclusion` doesn't work like `validate_required` so it will return a valid changeset if there is no :max_price